sites/all/modules/custom/training/templates/training_template.tpl.php

```diff
@@ -1,6 +1,23 @@
+<?php
+
+/**
+ * @file
+ * Displays a page element (list)
+ * Called from function training_theme()
+ *
+ * Available variables (created in function training_theme_page):
+ * - $title: List title
+ * - $links array: Links
+ * - $items array: Names of link
+ */
+?>
+
 <ul><strong><?php echo $title ?></strong>
-  <li><a href="<?php echo $links[0]?>"><?php echo $items[0]?></a></li>
-  <li><a href="<?php echo $links[1]?>"><?php echo $items[1]?></a></li>
-  <li><a href="<?php echo $links[2]?>"><?php echo $items[2]?></a></li>
-  <li><a href="<?php echo $links[3]?>"><?php echo $items[3]?></a></li>
-</ul>
+
+<?php
+  $list = '';
+  foreach ($links as $name=>$link){
+    $list .= '<li><a href="' . $link . '">' . $name . '</a></li>';
```

Comment:
Generate the links using https://api.drupal.org/api/drupal/includes%21common.inc/function/l/7.x